example_config_class.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class ThingWorker
+  def self.process(message)
+    FastlyNsq.logger.info("IN PROCESS: #{message}")
+  end
+end
+
+FastlyNsq.configure do |config|
+  config.channel = 'william'
+  # config.logger = Logger.new
+  # config.preprocessor = ->(message) { FastlyNsq.logger.info 'PREPROCESSESES' }
+
+  config.listen_to do |topics|
+    topics.add('assign_supplemental_plans', ThingWorker)
+  end
```

Comment:
I like the idea that one could configure multiple processors for a single topic this way, but if it also requires an additional channel, that seems to undercut the utility somewhat.

It occurs to me that there are two configure blocks -- `configure do |config|` and also `listen_to do |topics|` -- that could possibly be made more elegant with just one -- possibly `config.listen_to topic, Processor`.